server/src/main/java/org/apache/iotdb/db/engine/snapshot/SnapshotTaker.java

```diff
@@ -117,83 +208,150 @@ public boolean takeFullSnapshot(String snapshotDirPath, boolean flushBeforeSnaps
     } finally {
       manager.readUnlock();
     }
+    return true;
+  }
+
+  private boolean createSnapshotInAnotherDisk(File snapshotDir) {
+    for (String dataDir : IoTDBDescriptor.getInstance().getConfig().getDataDirs()) {
+      File seqDir =
+          new File(
+              dataDir,
+              "sequence"
+                  + File.separator
+                  + dataRegion.getStorageGroupName()
+                  + File.separator
+                  + dataRegion.getDataRegionId());
```

Comment:
Can this function be combined with `createSnapshotInLocalDisk`？
actually, I think the only difference between the two functions is the target path of the snapshot dir.

Reply:
The core of these two functions to create hard links steps are the same, and this step has been extracted as function "createFileSnapshotToTargetOne".